wit/wasmcloud/lattice-control/lattice-control.wit

```diff
@@ -0,0 +1,560 @@
+package wasmcloud:lattice-control;
+
+/// This interface represents the functions necessary
+/// to implement the [lattice-control wasmCloud interface](https://github.com/wasmCloud/interfaces/blob/main/lattice-control/lattice-control-interface.smithy)
```

Comment:
Isn't this interface now the "canonical" one? Not sure if it will be confusing to link back to the smithy version

Reply:
Well this WIT interface is a translation of the Smithy one, I think we'd organize it differently if we were doing it from scratch. 

The interface we're implementing here *is* the smithy one, because it needs to stay backwards compatible -- I can't add/remove any fields.